src/C-DEngine/C-DEngines/C-DNMIService/cde_NMIAPI.cs

```diff
@@ -3252,6 +3252,16 @@ public static bool IsInitialized()
             return false;
         }
 
+        /// <summary>
+        /// returns true if the NMI UX is initialized
+        /// </summary>
+        /// <returns></returns>
+        public static bool IsUXInitialized()
```

Comment:
This might get confused with the IsUXInitialized flag in TheThingBase. Can we change this to "IsNMIUXInitialized"?  I thought we have this already as in "IsNMIModelReady"?

Reply:
Looks like this was a side effect of the instance creation race condition in the networking plug-in. Removed.